ipinfo/handler_async.py

```diff
@@ -47,38 +50,30 @@ def __init__(self, access_token=None, **kwargs):
         self.access_token = access_token
 
         # load countries file
-        self.countries = handler_utils.read_json_file(
-            kwargs.get("countries_file")
-            if kwargs.get("countries_file")
-            else COUNTRY_FILE_DEFAULT
+        self.countries = (
+            read_json_file(kwargs.get("countries_file")) or countries
```

Comment:
What I meant was, we wanna change it so it's not accepting a file from the user, but just an in-memory map. Same for the others.

That requires changing the name of the kwarg to not have the `_file` suffix and actually guess it also requires a major version update as it's now backwards incompatible.